src/snowflake/connector/result_batch.py

```diff
@@ -270,13 +270,20 @@ def _download(self, **kwargs) -> "Response":
         for retry in range(MAX_DOWNLOAD_RETRY):
             try:
                 with TimerContextManager() as download_metric:
+                    logger.debug(
+                        "started downloading result batch of size %d", self.rowcount
+                    )
                     response = requests.get(
                         self._remote_chunk_info.url,
                         headers=self._chunk_headers,
                         timeout=DOWNLOAD_TIMEOUT,
                         stream=True,
                     )
                     if response.ok:
+                        logger.debug(
+                            "successfully downloaded result batch of size %d",
+                            self.rowcount,
```

Comment:
This type of logging is error prone, let's use f-strings instead
```suggestion
                            f"successfully downloaded result batch of size {self.rowcount}",
```